kiwi/repository/dnf.py

```diff
@@ -253,9 +263,9 @@ def _create_runtime_config_parser(self):
         self.runtime_dnf_config.set(
             'main', 'gpgcheck', self.gpg_check
         )
-        self.runtime_dnf_config.set(
-            'main', 'repo_gpgcheck', self.gpg_check
-        )
+        # We are not setting repo_gpgcheck, which forces repository
+        # signatures checks, because current Fedora 25 releases do not
+        # provide signed repositories, only signed packages
```

Comment:
Yes you are right, I was in doubt about that, as it is not a problem of DNF but Fedora (the package manager could be used even in different distros a part form Fedora). But I could  not find a clear way to check it, so I preferred not to include it. Also if Fedora does not sign its main repos the whole singatures check would have to be disabled if the main repo is included (which is likely to happen). We are not setting the signature checks on repo basis, so in that case, I assume checking package signatures already better than nothing.